force-app/main/default/classes/DonationHistoryController.cls

```diff
@@ -73,11 +77,37 @@ public with sharing class DonationHistoryController {
             throw new AuraHandledException(ex.getMessage());
         }
     }
+    
+    /** 
+    * @author Salesforce.org
+    * @date 2021
+    * @description method to return whether or not Payments are enabled in org
+    * @return boolean isPaymentEnabled
+    */
+    @AuraEnabled(cacheable=true)
+    public static boolean arePaymentsEnabled(){
+        npe01__Contacts_And_Orgs_Settings__c settings = npe01__Contacts_And_Orgs_Settings__c.getOrgDefaults();
+		boolean isPaymentEnabled = settings.npe01__Payments_Enabled__c;
+		return isPaymentEnabled;
```

Comment:
i think this code has to be in a service because it is business login.
Also, code could be lot easier:

return npe01__Contacts_And_Orgs_Settings__c.getOrgDefaults().npe01__Payments_Enabled__c

Reply:
agreed